lib/data/woocommerce/repositories/woocommerce_wrapper.dart

```diff
@@ -7,6 +7,7 @@ import 'package:openflutterecommerce/data/error/exceptions.dart';
 
 abstract class WoocommercWrapperAbastract{
    Future<List<dynamic>> getCategoryList({int parentId=0});
+   Future<List<dynamic>> getProductList({int categoryId=0});
```

Comment:
all filter rules should be here as parameters

Reply:
Agree